stable/postgresql/templates/NOTES.txt

```diff
@@ -1,41 +1,53 @@
-PostgreSQL can be accessed via port 5432 on the following DNS name from within your cluster:
-{{ template "postgresql.fullname" . }}.{{ .Release.Namespace }}.svc.cluster.local
+{{- if contains .Values.service.type "LoadBalancer" }}
+{{- if not .Values.postgresqlPassword }}
+-------------------------------------------------------------------------------
+ WARNING
 
-{{- if .Values.existingSecret }}
-If you have not already created the postgres admin secret:
+    By specifying "serviceType=LoadBalancer" and not specifying "postgresqlPassword"
+    you have most  likely exposed the PostgreSQL service externally without any
+    authentication mechanism.
 
-   kubectl create secret generic {{ .Values.existingSecret }} --namespace {{ .Release.Namespace }} --from-file=./postgres-password
-{{ else }}
-To get your user password run:
+    For security reasons, we strongly suggest that you switch to "ClusterIP" or
+    "NodePort". As alternative, you can also specify a valid password on the
+    "postgresqlPassword" parameter.
 
```

Comment:
We should add something about how to connect when the networkpolicy is enabled (it was in the previous notes). Not sure if I missed it